helper/TabManager.php

```
@@ -40,21 +42,21 @@ public function renderPanels($resource, $contentRegion = 'main', $layout = 'vert
         ]);
     }
 
-    public function renderTabsOnly($resource, $contentRegion = 'main')
+    public function renderTabsOnly($resource, $contentRegion = 'main', $layout = 'vertical') 
```

PHPMD (2.15.0)

Avoid unused parameters such as '$layout'. (Unused Code Rules)

(UnusedFormalParameter)